FrejaEidClient/src/main/java/com/verisec/frejaeid/client/client/impl/OrganisationIdClient.java

```diff
@@ -16,13 +16,18 @@
 import javax.net.ssl.SSLContext;
 import com.verisec.frejaeid.client.exceptions.FrejaEidClientPollingException;
 import com.verisec.frejaeid.client.exceptions.FrejaEidException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * Performs actions with organisation eID.
  *
  */
 public class OrganisationIdClient extends BasicClient implements OrganisationIdClientApi {
 
+    public static final Logger LOG = LoggerFactory.getLogger(OrganisationIdClient.class);
+    private static final long DEFAULT_EXPIRY_TIME_IN_MILLIS = 604800000;
```

Comment:
Can we use TimeUnit for this? Or just to write it as multiplication of day, hours, minutes, seconds and milliseconds, I think it's more readable this way?

Reply:
Used time units.